packages/cross_file/lib/src/types/base.dart

```diff
@@ -44,6 +44,8 @@ abstract class XFileBase {
   }
 
   /// For web, it may be necessary for a file to know its MIME type.
+  ///
+  /// For io implementation, this is not intended to be set and will be `null`.
```

Comment:
This isn't accurate; it doesn't have to be set, but it can be; there's no guarantee that it will be null.